opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -442,6 +442,9 @@ namespace Opm
             }
         }
 
+        std::string simulationDate(std::size_t currentStep) const;
```

Comment:
These small utilities do not deserve to become members in the already crowded Schedule class - just make anonymous free functions.